### ISSUE TITLE
Kafka/Jobs monitoring

### ISSUE THREAD

**Issue by mensfeld** [shellandbull/dockerization#2]
FYI Karafka has a web-ui for this that does not require any third party DB: https://karafka.io/docs/Web-UI-Getting-Started/

Cheers!

**Comment by shellandbull** [comment]
Awesome! This app isn't about that but I'll definitely be adding that onto this project :-)

**Comment by mensfeld** [comment]
Closing then :) don't hesitate to ping me if you need anything

**Comment by shellandbull** [comment]
@mensfeld actually now that you're here and I'm troubleshooting the installation, would you happen to know why my installation of the web UI fails?

```
bash-5.1# bundle exec karafka-web install
bundler: failed to load command: karafka-web (/usr/local/bundle/bin/karafka-web)
/usr/local/bundle/gems/karafka-rdkafka-0.13.5/lib/rdkafka/metadata.rb:33:in `initialize': Local: Broker transport failure (transport) (Rdkafka::RdkafkaError)
```

I understand is a connection issue to the kafka server, but running a manual sample job(`ExampleJob`) does succeed.

**Comment by mensfeld** [comment]
@shellandbull is it over the wire?

**Comment by mensfeld** [comment]
that is, do you connect to a remote kafka cluster?

**Comment by shellandbull** [comment]
@mensfeld nope, its all via `docker-compose up --build`. All the code and infra are on this repo

**Comment by mensfeld** [comment]
@shellandbull sounds like a misconfiguration where particular pieces of karafka or librdkafka cannot reach kafka. It's a common issue with docker (many articles were written about it). Would you mind joining our slack channel? So we could coop easier: https://slack.karafka.io/ Im sure we can solve this out.
